include/aie/Dialect/AIEX/IR/AIEXDialect.h

```diff
@@ -13,6 +13,12 @@
 
 #include "aie/Dialect/AIE/IR/AIEDialect.h"
 
+#include "mlir/IR/BuiltinAttributes.h"
+
+namespace xilinx::AIEX {
+const char defaultRuntimeSequenceName[] = "sequence";
```

Comment:
Could this be in the op like `RuntimeSequenceOp::defaultName` and still work in tablegen?
Or better,
```c++
let extraClassDeclaration = [{
     // Get the attributes name
     llvm::StringRef getDefaultName() { return "sequence"; }
}];
```